scss/pages/_inner.scss

```diff
@@ -0,0 +1,65 @@
+.post-content {
```

Comment:
Maybe `_content` will be a better file name? Or `_post`?

Reply:
i'll change file name in the pr#74